extensions/wikia/PowerUser/PowerUser.class.php

```diff
@@ -12,9 +12,13 @@
 namespace Wikia\PowerUser;
 
 use Wikia\Logger\Loggable;
+use Wikia\DependencyInjection\Injector;
+use Wikia\Service\User\Permissions\PermissionsAccessor;
+use Wikia\Service\User\Permissions\PermissionsService;
 
 class PowerUser {
 	use Loggable;
+	use PermissionsAccessor;
```

Comment:
The name of the trait doesn't suggest you will access a service, how about renaming it to `PermissionsServiceAccessor` ?